component/evaluationCard.jsx

```diff
@@ -0,0 +1,64 @@
+  background-size: 80%;
+  margin-bottom: 2rem;
+  min-height: 29.8rem; }
+  @media only screen and (max-width: 48em) {
+    .plaint-card {
+      background-size: 60%; } }
+  @media only screen and (max-width: 36em) {
+    .plaint-card {
+      background-size: 40%; } }
+  .plaint-card:hover .pc-title {
+    color: #545dd3; }
+  .plaint-card .illustration {
+    height: 14rem;
+    width: 100%;
+    border-radius: 2px;
+    margin-top: 2rem;
+    overflow: hidden; }
+    .plaint-card .illustration img {
+      height: 100%;
+      width: 100%;
+      object-fit: cover; }
+  .plaint-card .pc-title {
+    font-size: 1.5rem;
+    font-weight: 500;
+    margin-top: 1rem;
+    color: #68b1a7;
+    display: block; }
+  .plaint-card small {
+    display: block;
+    font-size: 1.3rem;
+    color: #b0b0bd;
+    font-weight: 300; }
+  .plaint-card .badge {
+    color: #b0b0bd;
+    background-color: #171721;
+    padding: 5px 10px;
+    font-size: 1.3rem;
+    border-radius: 4px; }
+`);
+
+export const EvaluationCard = ({data}) => {
+  return <Link href={`/evaluation/${data.id}`}>
+  <div class="plaint-card">
+      <span class="badge">{data.name}</span>
+      <div class="illustration">
+          <Img source={"illustrations/"+data.image} alt="illustration" />
```

Comment:
you can use backtiks tips (``) instead of concatenation word to keep the code clean
```jsx
<Img source={`illustations/${data.image}`}  alt="illustration"/>
```